openml/tasks/functions.py

```diff
@@ -327,27 +326,39 @@ def get_tasks(
 
 @openml.utils.thread_safe_if_oslo_installed
 def get_task(
-    task_id: int, download_data: bool = True, download_qualities: bool = True
+    task_id: int, *dataset_args, download_splits: Optional[bool] = None, **get_dataset_kwargs
 ) -> OpenMLTask:
     """Download OpenML task for a given task ID.
 
-    Downloads the task representation, while the data splits can be
-    downloaded optionally based on the additional parameter. Else,
-    splits will either way be downloaded when the task is being used.
+    Downloads the task representation, while the data splits can be downloaded optionally based on
+    the additional parameter that are passed to :meth:`openml.datasets.get_dataset`.
```

Comment:
I don't quite understand this. If the `download_splits` is given, this has nothing to do with the additional parameter, or does it?

Reply:
I will update the doc string to make it more clear.